google-cloud-spanner/src/main/java/com/google/cloud/spanner/connection/Connection.java

```diff
@@ -968,11 +969,29 @@ default RpcPriority getRPCPriority() {
    * the statement. {@link com.google.cloud.spanner.ReadContext.QueryAnalyzeMode#PROFILE} executes
    * the DML statement, returns the modified row count and execution statistics, and the effects of
    * the DML statement will be visible to subsequent operations in the transaction.
+   *
+   * @deprecated Use {@link #analyzeUpdateStatement(Statement, QueryAnalyzeMode, UpdateOption...)}
```

Comment:
```suggestion
   * @deprecated Use {@link #analyzeUpdateStatement(Statement, QueryAnalyzeMode, UpdateOption...)}
                instead.
```

Reply:
Good point, done. (note, Javadoc block elements that only contain one sentence should not end with a punctuation mark)